tests/utils/MockZkClient.js

```diff
@@ -0,0 +1,11 @@
+class MockZkClient {
```

Comment:
I feel like we should keep this mock solely in the ingestion reader test file because I don't know if this could be re-used by other test components